exercises/run-length-encoding/test/Tests.hs

```diff
@@ -0,0 +1,75 @@
+{-# LANGUAGE RecordWildCards #-}
+
+import Data.Foldable     (for_)
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+
+import RunLength (encode, decode)
+
+main :: IO ()
+main = hspecWith defaultConfig {configFastFail = True} specs
+
+specs :: Spec
+specs = describe "run-length-encoding" $ do
+          describe "decode" $ for_ decodeCases $ test decode
+          describe "encode" $ for_ encodeCases $ test encode
+          describe "both"   $ for_ bothCases   $ test (decode . encode)
+  where
+    test f Case{..} = it description $ f input `shouldBe` expected
+
+-- Test cases adapted from file
+-- `exercism/x-common/exercises/run-length-encoding/canonical-data.json`
+-- on 2016-12-26.
```

Comment:
This test suite could be better "translated" as a big `do` block, reproducing the exact test ordering from `x-common`.

That said, the cases ordering doesn't seem to be so important in this exercise, so I think it is great the way it is.